packages/utils/src/common/pickFalsy/internal.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  isNil,
+  isBoolean,
+  isNumber,
+  isString,
+  isArray,
+  isPlainObject,
+} from '../../validator';
+
+const isInvalidBoolean = (value: unknown) =>
+  isBoolean(value) && value === false;
+const isInvalidNumber = (value: unknown) =>
+  isNumber(value) && (Number.isNaN(value) || value === 0);
+const isInvalidString = (value: unknown) => isString(value) && value === '';
+
+const isInvalidArray = (value: unknown) => isArray(value) && value.length === 0;
+const isInvalidObject = (value: unknown) =>
+  isPlainObject(value) && Object.keys(value).length === 0;
+
+export const FALSY_MAPPER = {
+  nil: isNil,
+  boolean: isInvalidBoolean,
+  number: isInvalidNumber,
+  string: isInvalidString,
+  array: isInvalidArray,
+  object: isInvalidObject,
+} as const;
```

Comment:
```suggestion
export const FALSY_MAPPER = {
  null: isNil,
  undefined: isNil,
  boolean: isInvalidBoolean,
  number: isInvalidNumber,
  string: isInvalidString,
  array: isInvalidArray,
  object: isInvalidObject,
} as const;
```

기존 `undefined`, `null`을 `nil`로 하기보다 (누군가에게 헷갈릴 수 있을 것 같습니다..!)
key는 모두가 아는 네이밍인 undefined, null을 유지하돼, 함수를 isNil 사용을 제안드려봅니다!

Reply:
@ssi02014 좋은 의견 감사합니다! 제가 null, undefined에서 isNil로 통합했던 이유는 isNil은 결국 `value === null || value === undefined` 라는 점에서 내부적으로 현재는 typeToFunction Mapper를 통해서 선택된 타입을 체크함수로 바꿔주는데, null과 undefined이 모두 isNil을 사용하게된다면 value가 undefined인 경우에 물론 falsy로 걸리겠지만 순서상 먼저 체크되어 종료된다는 점에서 살짝 걸렸었네요! 😂 

다만 이는 실제 검증의 관점에서는 문제가 없기때문에 말씀주신 방법으로 수정해두겠습니다! 👍👍 

```typescript
// value가 undefined인 경우 some함수가 실행되며 첫번째 validation의 목적이었던 null이 isNil로 치환되어 처음부터 로직 종료
['null', 'undefined', 'number', 'string'] => [isNil, isNil, isInvalidNumber, isInvalidString]
```

Reply:
@Sangminnn null과 undefined는 기존 논의대로 무조건 falsy 체크를 하기로 결정됐기 때문에 null과 undefined중 누가 먼저 실행되는지는 크게 문제가 없을 것 같습니다!! 🤗